testsuite/gnat2goto/tests/function_pointer/test.out

```diff
@@ -1,2 +1,25 @@
+Standard_Output from gnat2goto test:
```

Comment:
What's going on with these two tests? This looks like the tests are now producing missing feature reports which they were not before.... Were this test and procedure_pointer both passing before this change?

Reply:
It’s basically just Itypes that we never supported, but we just “happened” to pass these tests without missing feature reports or crashes because we don’t actually _use_ these types anywhere in the resulting goto program.

I.e. as a side effect of some of the changes we’ve made in this PR some things that didn’t work before now _visibly_ don’t work.

Reply:
OK. There's recently been some debate on https://github.com/diffblue/gnat2goto/pull/327 about whether we should or should not add "missing feature" reports to expected outputs - I'm not sure we came to a conclusion. In this case, it's probably better to accept it in the expected output and keep the test passing, rather than turn a passing test into an xfail.